indico/modules/categories/util.py

```diff
@@ -129,6 +130,7 @@ def get_category_stats(category_id=None):
 
 @memoize_redis(3600)
 @materialize_iterable()
+@make_interceptable
```

Comment:
are you sure you don't want it one above so you get the list instead of an iterable?